libs/langchain-community/src/llms/watsonx_ai.ts

```diff
@@ -1,3 +1,4 @@
+/** @deprecated Please use ibm instead */
```

Comment:
A better message here would be nice - also I think you have to put this comment on top of the class